db/migrate/20200624185243_create_review_items.rb

```diff
@@ -0,0 +1,20 @@
+class CreateReviewItems < ActiveRecord::Migration[6.0]
+  def change
+    create_table :review_items do |t|
+      t.references :reviewable, polymorphic: true, null: false, index: { name: :index_on_reviewable_type_and_id }
+      t.references :reviewer, references: :users, foreign_key: { to_table: :users }
+      t.boolean :reviewed, null: false, default: false
+      t.boolean :read, null: false, default: false
+      t.string :action_taken
```

Comment:
Doesn't appear to be being used yet, but we probably want to validate inclusion of whatever `action_taken` is :)